overlays/nlp-engine/vcpkg.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nlp-engine",
+  "version-semver": "1.21.7",

```

Comment:
@GordonSmith is this the NLP Engine version? If so, it would need to be updated.

Reply:
Yes.